### PR TITLE
feature: add support for Raymarine SeaTalk-STNG-Converter AP PGNs

### PR DESCRIPTION
**Pull request by godind**
Support for proprietary PGNs of Raymarine SeaTalk-STNG-Converter connected to Raymarine SeaTalk SmartPilot (S1, S2, S3) course computer with ST600X control heads.

I tried to filter 126720 as much as possible with PGN description "Seatalk1: Pilot Mode" and Manufacturer "Raymarine" to limit potential collision with other devices - such as FUSION radio and others.

65359 is the PGN SeaTalk converted Pilot Heading.,

### CODE REVIEW

File: raymarine/126720.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        return 'wind';
+      }
+      else if ( (mode == 74  && subMode == 0 ) {
+        return 'route';
+      }
+      else if ( mode == 66 && ( subMode == 0 || subMode == 4 ) ) { //subMode 4 means offcourse
+        return 'auto';
+      }
+      else {
+        console.log('n2k-signalk 126720.js undefined AP state found. n2k data: ' + JSON.stringify(n2k))
```
Comment (sbender9) [inline raymarine/126720.js:30]: I think I'd rather a return value like 'unknown' instead of flooding the log.

**Review by sbender9 — CHANGES_REQUESTED** [review]

**Comment by sbender9** [comment]
This need tests.

File: raymarine/126720.js (hunk 1 above)
Comment (godind) [reply raymarine/126720.js]: Is this update better ?

**Comment by godind** [comment]
I think this should do it. Let me know @sbender9 if I need to fix something else. Thanks

**Pull request merged**